rustsec/Cargo.toml

```
@@ -39,7 +36,7 @@ serde_json = "1"
 
 [features]
 default = ["git"]
```

With no additional dependencies, `fix` can probably be enabled by default:

```suggestion
default = ["fix", "git"]
```

---

In that case should I remove the feature outright and just make it always enabled?